katas/design-patterns/constructor-pattern/fruitexample.js

```diff
@@ -0,0 +1,16 @@
+class Fruit {
+    constructor(name, taste) {
```

Comment:
space after the constructor word

Reply:
Same here as i said before, eslint doesn't require you to use space between the function declaration and the parenthesis